h2o-extensions/xgboost/src/main/java/hex/tree/xgboost/XGBoost.java

```diff
@@ -438,13 +442,94 @@ private void scoreAndBuildTrees(final XGBoostSetupTask setupTask, final BoosterP
           break;
         }
       }
+
+      Map<String, Integer> monotoneConstraints = _parms.monotoneConstraints();
+      if ((!monotoneConstraints.isEmpty()) && constraintCheckEnabled()) {
+        checkConstraints(model.model_info(), monotoneConstraints);
```

Comment:
It would be nice and reassuring to have a Log message that says that the constraints were successfully checked.

Reply:
added _job.update